assets/js/util/index.js

```diff
@@ -573,28 +586,26 @@ export const decodeHTMLEntity = ( str ) => {
  * Get the icon for a module.
  *
  * @param {string}  module                The module slug.
- * @param {boolean} blockedByParentModule Whether the module is blocked by a parent module.
  * @param {string}  width                 The icon width.
  * @param {string}  height                The icon height.
  * @param {string}  useClass              Class string to use for icon.
  *
  * @return {HTMLImageElement}             <img> tag with module icon.
  */
-export function moduleIcon( module, blockedByParentModule, width = '33', height = '33', useClass = '' ) {
-	if ( ! global._googlesitekitLegacyData ) {
+export function moduleIcon( module, width = '33', height = '33', useClass = '' ) {
+	if ( ! global._googlesitekitLegacyData || ! moduleMap.hasOwnProperty( module ) ) {
 		return;
 	}
 
-	/* Set module icons. Page Speed Insights is a special case because only a .png is available. */
-	let iconComponent = <SvgIcon id={ module } width={ width } height={ height } className={ useClass } />;
+	const ModuleIconComponent = moduleMap[ module ];
 
-	if ( blockedByParentModule ) {
-		iconComponent = <SvgIcon id={ `${ module }-disabled` } width={ width } height={ height } className={ useClass } />;
-	} else if ( 'pagespeed-insights' === module ) {
-		iconComponent = <img src={ global._googlesitekitLegacyData.admin.assetsRoot + 'images/icon-pagespeed.png' } width={ width } alt="" className={ useClass } />;
-	}
-
-	return iconComponent;
+	return (
+		<ModuleIconComponent
+			width={ module === 'pagespeed-insights' ? undefined : width }
```

Comment:
@aaemnnosttv I think accepting the visual change is the fix for this one as all of the other icons behave similarly (width of `33px` with a variable height) whereas the PageSpeed icon had a width of `38.22px`


**"Broken" PageSpeed Icon, previous implementation (38.22px width)**
![image](https://user-images.githubusercontent.com/8092736/95212570-b05e5280-07a2-11eb-84f9-fe8352ba6243.png)

**PageSpeed Icon with matching behavior (33px width)**
![image](https://user-images.githubusercontent.com/8092736/95212771-e56aa500-07a2-11eb-97fc-84a9e8aa55d2.png)

Reply:
@caseymorrisus this looks okay to me but since this is a user-facing change I'd like to get @felixarntz to weigh in on whether or not this is okay to change.

I think `moduleIcon`s should have consistent dimensions, regardless of which icon is shown. I think the reason the PSI icon is shown larger is to compensate for it's content being shorter vertically which gives the appearance of it looking smaller than others. In this case, you could argue that perhaps the `SetupModule` component (I think it is) that calls `moduleIcon` for PSI specifically could call it with the larger width to compensate, but I think `moduleIcon` itself should be consistent.

Reply:
@aaemnnosttv I had the same thought process when comparing the Analytics icon & PageSpeed icon side by side, but if you compare them to the other icons they're all similar in size, that's my bad for not providing a better screenshot. The Analytics icon is the outlier, if anything it should be the element to get a 1-off style to match the other 3 icons.

See this screenshot below (hid the Analytics section) to see how the icons compare with each other when they all have matching behavior (no conditionals):

![image](https://user-images.githubusercontent.com/8092736/95340458-0b0eb180-086a-11eb-97fe-735d2f4f80e4.png)

Reply:
I think the new format is okay, let's keep the module icon widths consistent. I had some related questions in my PR review, like why aren't the dimensions for the module icons always square?